File: demo-test/src/test/java/io/fries/demo/test/CucumberTestRunner.java
package io.fries.demo.test;

import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;
import org.junit.runner.RunWith;
import org.springframework.context.annotation.ComponentScan;

@RunWith(Cucumber.class)
@CucumberOptions(
        glue = "io.fries.demo.test.cucumber",
        features = "src/test/resources/features",
        plugin = {"pretty", "html:target/cucumber/cucumber.html", "json:target/cucumber/cucumber.json"}
)
public class CucumberTestRunner {
}

File: demo-test/src/test/kotlin/io/fries/demo/test/CucumberTestRunner.kt
package io.fries.demo.test

import io.cucumber.junit.Cucumber
import io.cucumber.junit.CucumberOptions
import org.junit.runner.RunWith

@RunWith(Cucumber::class)
@CucumberOptions(
    glue = ["io.fries.demo.test.cucumber"],
    features = ["src/test/resources/features"],
    plugin = ["pretty", "html:target/cucumber/cucumber.html", "json:target/cucumber/cucumber.json"]
)
class CucumberTestRunner